sklearn/metrics/tests/test_pairwise.py

```diff
@@ -407,6 +417,72 @@ def test_euclidean_distances():
     assert_greater(np.max(np.abs(wrong_D - D1)), .01)
 
 
+def test_masked_euclidean_distances():
```

Comment:
can we check that the result is the same than ``euclidean_distances`` if there is no missing values?